src/http-clients.md

```diff
@@ -1,4 +1,45 @@
 ---
 ---
 
-# Clients HTTP Interface
\ No newline at end of file
+# Clients HTTP Interface
+
+Details of a clients which will be accessing Learning Locker. It contains details for permissions, authenticating and storing the xapi request.
+
+It is accessible through the [connection API](/http-connection) and [restify api](/http-models).
+
+### Schema
+
+Name | Description
+--- | ---
+_id | The unique id of document.
+createdAt | When this document was created.
+updatedAt | When this document was last updated.
+title | String. The title of the client.
+api | The client [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) details. [See api](#api).
+authority | A json encoded string of an agent object that will be added to any statements the client pushes statement model. [See authority](#authority).
+scopes | An array of strings. permission [scopes](/http-roles#scopes) that this client has permission for.
+isTrusted | Whether this client is active. 
+organisation | The [organisation](/http-organisations#schema) this client is attached to.
+lrs_id | The id of the [store](/http-stores#schema) that the system will put recieved xapi statements.
+
+# api
+
+Name | Description
+--- | ---
+basic_key | The basic auth key.
+basic_secret | The basic auth secret.
+
+# authority
+
+This is a json encoded string, of the following properties:
+
+Name | Description
+--- | ---
+objectType | 'Agent'.
+name | Name of the agent.
+mbox | Optional, of its an mbox
+mbox_sha1sum | Optional, mbox sha1
+openid | Optional, the open id.
+homePage | Optional, the homePage.
```

Comment:
No `homePage` property. This is contained within the `account` property that is missing here. There is also a `name` property inside the `account` property.

Reply:
Commented.